tools/please_shim/main.go

```diff
@@ -27,10 +27,6 @@ var opts struct {
 		Profile  []core.ConfigProfile `long:"profile" env:"PLZ_CONFIG_PROFILE" env-delim:";" description:"Configuration profile to load; e.g. --profile=dev will load .plzconfig.dev if it exists."`
 	} `group:"Options controlling what to build & how to build it"`
 
-	OutputFlags struct {
-		Verbosity cli.Verbosity `short:"v" long:"verbosity" description:"Verbosity of output (error, warning, notice, info, debug)" default:"warning"`
```

Comment:
I wonder if you want to leave this in but change the name so it's still possible to run the shim in verbose mode if needed?

Reply:
Agreed.

Reply:
Oh, I do think you might need to rename it though if you do that - I think we often invoke plz in CI like `plz -p -v 2 build //blahblah` which would override this

Reply:
Ah, I see what you mean now. Will change.